extensions/blocks/revue/edit.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * WordPress dependencies
+ */
+import { BlockControls } from '@wordpress/block-editor';
+import {
+	Button,
+	Disabled,
+	IconButton,
+	Placeholder,
+	TextControl,
+	Toolbar,
+} from '@wordpress/components';
+import { useEffect, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import ButtonPreview from './button-preview';
+import { icon } from './';
+
+export default function RevueEdit( props ) {
+	const { attributes, className, setAttributes } = props;
```

Comment:
It might be nice to use [getValidatedAttributes](https://github.com/Automattic/jetpack/blob/b84c841b55eec0855ca5ed0b60cab8f8268cb989/extensions/shared/get-validated-attributes.js#L13) here so that we can ensure that users aren't breaking blocks by accidentally modifying attributes.

Reply:
Fixed in c608a48c37d89a1195f6fc280bc3ff21c6ca4400